### PR TITLE
Astrum refactor

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Reorganises Astrum file structure and code layout, as well as refactoring the abductor mobs into basic mobs.

## Proof of Testing
<details>
<summary>Screenshots/Videos</summary>
 
image

</details>

## Changelog
:cl:
refactor: Abductor mobs in the Astrum gateway (not the antagonist) have better AI.
/:cl:

### CODE REVIEW

**Comment by RatFromTheJungle** [comment]
add atleast one gun

**Review by Jolly-66 — APPROVED** [review]
maps fine

**Review by Zonespace27 — APPROVED** [review]
Step 1 of making astrum not completely awful to play through

File: modular_skyrat/modules/awaymissions_skyrat/mothership_astrum/gear.dm [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+/obj/item/gun/energy/alien/zeta
+	name = "Zeta Blaster"
+	desc = "Having this too close to your face makes you start to taste blood, is this safe?"
+	icon = 'modular_skyrat/modules/awaymissions_skyrat/icons/alienblaster.dmi'
+	lefthand_file = 'modular_skyrat/modules/awaymissions_skyrat/icons/alienhand.dmi'
+	righthand_file = 'modular_skyrat/modules/awaymissions_skyrat/icons/alienhand2.dmi'
+	icon_state = "alienblaster"
+	inhand_icon_state = "alienblaster"
+	pin = /obj/item/firing_pin
+	selfcharge = 1
```
Comment (Zonespace27) [inline modular_skyrat/modules/awaymissions_skyrat/mothership_astrum/gear.dm]: is selfcharge an int or a bool?

File: modular_skyrat/modules/awaymissions_skyrat/mothership_astrum/mob.dm [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+	attack_sound = 'sound/weapons/punch1.ogg'
+
+	combat_mode = TRUE
+	status_flags = CANPUSH
+	speed = 2
+
+	ai_controller = /datum/ai_controller/basic_controller/abductor
+
+	/// What this mob drops on death
+	var/loot = list(/obj/effect/gibspawner/generic, /obj/effect/spawner/random/astrum/sci_loot)
```
Comment (Zonespace27) [inline modular_skyrat/modules/awaymissions_skyrat/mothership_astrum/mob.dm]: typecast to a list and (maybe) static, will want to check how it works w/ subtypes just to be safe

File: modular_skyrat/modules/awaymissions_skyrat/mothership_astrum/mob.dm [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+/mob/living/basic/abductor/Initialize(mapload)
+	. = ..()
+	if(LAZYLEN(loot))
```
Comment (Zonespace27) [inline modular_skyrat/modules/awaymissions_skyrat/mothership_astrum/mob.dm]: this isn't a lazylist

File: modular_skyrat/modules/awaymissions_skyrat/mothership_astrum/mob.dm [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+	maxHealth = 1750
+	health_doll_icon = "pandora"
+	death_message = "falls to their knees before exploding into a ball of gore."
+
+	attack_verb_continuous = "attacks"
+	attack_verb_simple = "attack"
+
+/mob/living/simple_animal/hostile/megafauna/hierophant/astrum/bullet_act(obj/projectile/bullet)
+	apply_damage(bullet.damage, bullet.damage_type)
+	return // no more reduction
```
Comment (Zonespace27) [inline modular_skyrat/modules/awaymissions_skyrat/mothership_astrum/mob.dm]:
```suggestion
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
Actually just a few small things w/ old astrum code that i noticed

File: modular_skyrat/modules/awaymissions_skyrat/mothership_astrum/mob.dm (hunk 2 above)
Comment (tf-4) [reply modular_skyrat/modules/awaymissions_skyrat/mothership_astrum/mob.dm]: doesn't work with static but typecasts to a list fine

**Review by Zonespace27 — APPROVED** [review]
cool

**Pull request merged**